app/test_main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+import unittest
+from unittest import mock
+
+import app.main
+
+
+class TestCanAccessGooglePage(unittest.TestCase):
+    @mock.patch("app.main.has_internet_connection")
+    @mock.patch("app.main.valid_google_url")
+    def test_accessible(
+            self,
+            mock_valid_url: bool,
+            mock_has_internet: bool
+    ) -> None:
+        mock_valid_url.return_value = True
+        mock_has_internet.return_value = True
+        result = app.main.can_access_google_page("https://www.google.com")
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/220959619-cf280ba1-48cb-4ba0-985e-2ef473a0d47d.png)
also no need to change url in ur tests, u can write it as constant and use everywhere